components/infobox/wikis/rainbowsix/infobox_league_custom.lua

```diff
@@ -164,10 +157,8 @@ end
 
 function CustomLeague:defineCustomPageVariables()
 	-- Variables with different handling compared to commons
-	Variables.varDefine(
-		'tournament_liquipediatiertype',
-		Tier.text.types[string.lower(_args.liquipediatiertype or '')] or _DEFAULT_TIERTYPE
-	)
+	Variables.varDefine('tournament_liquipediatiertype',
```

Comment:
Should the default be set to the args already? Might mess with the tiertype list in MetdataGenerator tho.
Also is it missing in https://github.com/Liquipedia/Lua-Modules/blob/main/standard/tier/wikis/rainbowsix/tier_data.lua ?

Reply:
MetdataGenerator would be fine, but the display in infobox itself would get altered

Reply:
Oh right, yeah.